pkg/translator/api/ingress.go

```diff
@@ -61,7 +84,17 @@ func (o *IngressEdgeLBPoolSpec) Validate(obj *extsv1beta1.Ingress) error {
 	}
 	// Validate that the HTTP port is valid.
 	if err := validation.IsValidPortNum(int(*o.Frontends.HTTP.Port)); err != nil {
-		return fmt.Errorf("%d is not a valid port number", *o.Frontends.HTTP.Port)
+		return fmt.Errorf(".frontends.http.port %d is not a valid HTTP port number (valid range is between 1 and 65535)", *o.Frontends.HTTP.Port)
```

Comment:
would it be too much to move this validation functions to a validation.go file :) ?

Reply:
You mean the on below? Yes, I can do that.

Reply:
I mean the validation functions in this file :)